acceptance/suites/tests/010-puppetserver-cli/subcommand/SERVER-297_common_behavior.rb

```diff
@@ -13,6 +15,18 @@
   assert(env['JARS_NO_REQUIRE'], "JARS_NO_REQUIRE missing")
 end
 
+step "ruby: Check that proxy env-variables are present"
+on(master, "#{proxy_env_vars} puppetserver ruby -rjson -e 'puts JSON.pretty_generate(ENV.to_hash)'") do
+  env = JSON.parse(stdout)
+  assert(env['HTTP_PROXY'], "HTTP_PROXY is missing")
+  assert(env['http_proxy'], "http_proxy is missing")
+  assert(env['HTTPS_PROXY'], "HTTPS_PROXY is missing")
+  assert(env['https_proxy'], "https_proxy is missing")
+  assert(env['NO_PROXY'], "NO_PROXY is missing")
+  assert(env['no_proxy'], "no_proxy is missing")
+  assert_nil(env['RANDOM_VARIABLE'], "RANDOM_VARIABLE is present and should not be")
```

Comment:
There are some tests in the other subcommand specs that look at this, for example in the ![ruby.rb specs](https://github.com/puppetlabs/puppetserver/blob/a0b91a12d5760450448511dbe42c2e55df94b561/acceptance/suites/tests/010-puppetserver-cli/subcommand/ruby.rb#L18-L22).  If you also want to include one in the 'common_behavior' tests for that, I'd be good with it.  I'd agree, though, that maybe having it as a separate test step that sets `RANDOM_VARIABLE` without whitelisting it and confirming that it gets filtered out would be better.  Or if you just want to remove this line and not replace it with anything else, I'm good with that, too.